Flexa-Agents/Fallback-State/README.md

```diff
@@ -0,0 +1,19 @@
+# Fallback State Agent
+
+## Description
+
+This agent detects the contract entering a fallback state
```

Comment:
```suggestion
This agent detects when `FlexaCollateralManager` enters a fallback state
```
Suggestion for a slight re-wording as well as specifying which contract is being referred to.